### PR TITLE
Enable common verbosity arguments for pytest

### PR DESCRIPTION
**Pull request by Alexsandruss**
## Description

Changes:
- Enable common verbosity arguments for pytest calls in sklearnex testing scripts and store them in one plain text file
- Enable test duration tracking (top-100 tests with >50ms runtime)

---

PR should start as a draft, then move to ready for review state after CI is passed and all applicable checkboxes are closed.
This approach ensures that reviewers don't spend extra time asking for regular requirements.

You can remove a checkbox as not applicable only if it doesn't relate to this PR in any way.
For example, PR with docs update doesn't require checkboxes for performance while PR with any change in actual code should have checkboxes and justify how this code change is expected to affect performance (or justification should be self-evident).

Checklist to comply with **before moving PR from draft**:

**PR completeness and readability**

- [x] I have reviewed my changes thoroughly before submitting this pull request.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have updated the documentation to reflect the changes or created a separate PR with update and provided its number in the description, if necessary.
- [x] Git commit message contains an appropriate signed-off-by string _(see [CONTRIBUTING.md](https://github.com/uxlfoundation/scikit-learn-intelex/blob/main/CONTRIBUTING.md#pull-requests) for details)_.
- [x] I have added a respective label(s) to PR if I have a permission for that.
- [x] I have resolved any merge conflicts that might occur with the base branch.

**Testing**

- [x] I have run it locally and tested the changes extensively.
- [x] All CI jobs are green or I have provided justification why they aren't.
- [x] I have extended testing suite if new functionality was introduced in this PR.

**Performance**

N/A.

### CODE REVIEW

File: .ci/scripts/run_sklearn_tests.py [hunk 1]
```diff
@@ -48,8 +48,15 @@
...
         os.environ["SCIPY_ARRAY_API"] = "1"
 
+    pytest_verbosity_file = os.path.join(
+        os.path.dirname(os.path.dirname(os.path.dirname(__file__))),
+        ".pytest-verbosity-args",
+    )
+    with open(pytest_verbosity_file, "r") as f:
+        verbosity_args = f.read().strip()
+
     pytest_args = (
```
Comment (david-cortes-intel) [inline .ci/scripts/run_sklearn_tests.py:51]: This file now allows forwarding extra arguments to pytest by passing those arguments to this file when executing it. These verbosity arguments could be moved to the .sh file instead - that way, they wouldn't always be added unconditionally when this file is used locally outside of the CI scripts.

Comment (Alexsandruss) [reply .ci/scripts/run_sklearn_tests.py:51]: You are right, it also makes passing args much shorter.

**Review by david-cortes-intel — APPROVED** [review]
LGTM. But is there a way to propagate these results to the JSON files? They are quite hard to locate otherwise given the length of the logs.
